client/styles/components/_reset-password.scss

```diff
@@ -1,29 +1,13 @@
-.reset-password {
-  @extend %modal;
-  text-align: center;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  padding: #{20 / $base-font-size}rem;
-  align-items: center;
-}
-
-.reset-password-form__email-input {
-  width: #{300 / $base-font-size}rem;
-}
-
-.reset-password-form__field {
-  margin: #{20 / $base-font-size}rem 0;
-}
-
 .reset-password__submitted {
-  width: #{300 / $base-font-size}rem;
+  width: #{360 / $base-font-size}rem;
   display: none;
   text-align: left;
-  font-size: #{12 / $base-font-size}rem;
+  font-size: #{11.65 / $base-font-size}rem;
```

Comment:
why 11.65 pixels? I think most browsers will round to the nearest pixel anyway...